### ISSUE TITLE
Notation: pointed category

### ISSUE THREAD

**Issue by mauroporta** [mauroporta/Autour_de_DAG#7]
I'm working in fixing the notations of Pietro's _Algebraic structure in higher categories_, and I discovered that he uses C_\bullet to denote the pointed category of C.

I don't know whether other notations for the same notion are used throughout the other chapters, but I strongly suggest to use \* instead of \bullet. It will be consistent with my exposé and with Lurie's notation.

**Comment by pbelmans** [comment]
Yes, definitely. Bullets are for homological indexing. Of course, asterisks are used for simplicial indexing too, but at least it will be consistent with the other exposés and Lurie (which of course is the bible, according to you ;)).

**Comment by mauroporta** [comment]
Well not only Lurie... also Hirschhorn and, I believe, Hovey... :)
